src/app/api/public/[contractName]/[usersContractVersion]/sendTicket/route.tsx

```diff
@@ -85,38 +88,31 @@ async function postHandler(req: NextRequestWithApiTokenAuth, { params: { contrac
 
   await provider.waitForTransaction(transactionResult?.txHash);
 
-  // 🏗️ TODO: send email to ticket's receiver
-  // const transporter = await createMailTransport(isLocalhost);
-  // const testResult = await transporter.verify();
-  // if (!testResult) {
-  //   throw new Error("Email service is not ready");
-  // }
-  // const emailHtml = await render(<EmailSample />);
-  //
-  // const options = {
-  //   from: process.env.SMTP_EMAIL || "test@blessed.fan",
-  //   to: 'leeszczuu@gmail.com',
-  //   subject: "Your ticket!",
-  //   html: emailHtml,
-  // };
-  //
-  // const sendResult = await transporter.sendMail(options);
-  //
-  // if (isLocalhost) {
-  //   console.log(`📨 Email sent. Preview URL: ${nodeMailer.getTestMessageUrl(sendResult)}`);
-  // }
+  const transporter = await createMailTransport(isLocalhost);
+  const testResult = await transporter.verify();
```

Comment:
`createMailTransport` already  do verify transporter so we can skip 92 and next lines